backend/app.py

```diff
@@ -142,17 +145,21 @@ def quote():
 @app.route("/hostname")
 def hostname():
     """return the hostname of the given container"""
-    return jsonify({"backend": socket.gethostname()})
+    backend_hostname = socket.gethostname()
+    db_hostname = db.get_db_hostname(DB_CONN)
```

Comment:
Will this get us into trouble when DB is not connected? @zanderhavgaard

Reply:
Might need  a wrapper like: if check_if_db_is_available():

Reply:
Good catch, fixed in last commit :+1: